src/pybind/mgr/dashboard/frontend/e2e/rgw/bucketCreate.e2e-spec.ts

```diff
@@ -0,0 +1,212 @@
+import { $, browser, protractor } from 'protractor';
+import { Helper } from '../helper.po';
+import { BucketsCreatePage, BucketsPage } from './bucketCreate.po';
+
+describe('RGW Buckets Page', () => {
+  let page: BucketsPage;
+
+  beforeAll(() => {
+    page = new BucketsPage();
+  });
+
+  describe('Click Create to go to the Create page', () => {
+    beforeEach(() => {
+      page.navigateTo();
+    });
+
+    it('Should click the Create button', () => {
+      // Checks the breadcrumb at the Buckets page, clicks the create button and then
+      // checks the breadcrumb again to see if we moved to the Create page
+      page.navigateTo();
+      expect(Helper.getBreadcrumbText()).toEqual('Buckets');
+      page.getCreateABucket().click();
+      expect(Helper.getBreadcrumbText()).toEqual('Create');
+    });
+  });
+});
+
+describe('Create bucket from the Create page', () => {
+  let page: BucketsCreatePage;
+
+  beforeAll(() => {
+    page = new BucketsCreatePage();
+  });
+
+  describe('Types name, sets owner, and creates bucket', () => {
+    beforeEach(() => {
+      page.navigateTo();
+    });
+
+    it('Heading should be Create Bucket', () => {
```

Comment:
The `it `-sections usually start with an 'should [...]' in the beginning, could you please adapt this line as well as line [73](https://github.com/ceph/ceph/pull/28533/files#diff-c079219b5d7f8d1508461113218c595cR73)?